src/main/typescript/node_modules/@atomist/rug/operations/Handlers.ts

```diff
@@ -23,7 +23,7 @@ interface Instruction<T extends InstructionKind> {
 }
 
 class Respondable<T extends InstructionKind> {
-  instruction: Instruction<T>
+  instruction: Review | Edit | Generate
```

Comment:
Yes, I addressed the ability to put all instruction types in a plan, and exexcutes are indeed Respondables again.